2024-06-19 20:41:43,469 INFO    MainThread:2805280 [wandb_setup.py:_flush():76] Current SDK version is 0.15.12
2024-06-19 20:41:43,469 INFO    MainThread:2805280 [wandb_setup.py:_flush():76] Configure stats pid to 2805280
2024-06-19 20:41:43,469 INFO    MainThread:2805280 [wandb_setup.py:_flush():76] Loading settings from /user/as6154/.config/wandb/settings
2024-06-19 20:41:43,469 INFO    MainThread:2805280 [wandb_setup.py:_flush():76] Loading settings from /user/as6154/dissert/wandb/settings
2024-06-19 20:41:43,469 INFO    MainThread:2805280 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-06-19 20:41:43,469 INFO    MainThread:2805280 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-06-19 20:41:43,469 INFO    MainThread:2805280 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-06-19 20:41:43,469 INFO    MainThread:2805280 [wandb_init.py:_log_setup():528] Logging user logs to /user/as6154/dissert/wandb/run-20240619_204143-huj8u14g/logs/debug.log
2024-06-19 20:41:43,470 INFO    MainThread:2805280 [wandb_init.py:_log_setup():529] Logging internal logs to /user/as6154/dissert/wandb/run-20240619_204143-huj8u14g/logs/debug-internal.log
2024-06-19 20:41:43,470 INFO    MainThread:2805280 [wandb_init.py:init():568] calling init triggers
2024-06-19 20:41:43,470 INFO    MainThread:2805280 [wandb_init.py:init():575] wandb.init called with sweep_config: {}
config: {}
2024-06-19 20:41:43,471 INFO    MainThread:2805280 [wandb_init.py:init():593] re-initializing run, found existing run on stack: s2e60ynr
2024-06-19 20:41:43,472 INFO    MainThread:2805280 [wandb_run.py:_finish():1934] finishing run hong-exploration/GPT_QFT/s2e60ynr
2024-06-19 20:41:43,473 INFO    MainThread:2805280 [jupyter.py:save_history():445] not saving jupyter history
2024-06-19 20:41:43,473 INFO    MainThread:2805280 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-06-19 20:41:43,473 INFO    MainThread:2805280 [wandb_init.py:_jupyter_teardown():456] cleaning up jupyter logic
2024-06-19 20:41:43,473 INFO    MainThread:2805280 [wandb_run.py:_atexit_cleanup():2168] got exitcode: 0
2024-06-19 20:41:43,474 INFO    MainThread:2805280 [wandb_run.py:_restore():2151] restore
2024-06-19 20:41:43,475 INFO    MainThread:2805280 [wandb_run.py:_restore():2157] restore done
2024-06-19 20:41:51,888 INFO    MainThread:2805280 [wandb_run.py:_footer_history_summary_info():3599] rendering history
2024-06-19 20:41:51,888 INFO    MainThread:2805280 [wandb_run.py:_footer_history_summary_info():3631] rendering summary
2024-06-19 20:41:51,895 INFO    MainThread:2805280 [wandb_run.py:_footer_sync_info():3558] logging synced files
2024-06-19 20:41:51,959 INFO    MainThread:2805280 [wandb_init.py:init():618] starting backend
2024-06-19 20:41:51,959 INFO    MainThread:2805280 [wandb_init.py:init():622] setting up manager
2024-06-19 20:41:51,960 INFO    MainThread:2805280 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-06-19 20:41:51,960 INFO    MainThread:2805280 [wandb_init.py:init():628] backend started and connected
2024-06-19 20:41:51,968 INFO    MainThread:2805280 [wandb_run.py:_label_probe_notebook():1276] probe notebook
2024-06-19 20:41:51,969 INFO    MainThread:2805280 [wandb_run.py:_label_probe_notebook():1286] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-06-19 20:41:51,969 INFO    MainThread:2805280 [wandb_init.py:init():720] updated telemetry
2024-06-19 20:41:51,969 INFO    MainThread:2805280 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-06-19 20:41:53,067 INFO    MainThread:2805280 [wandb_run.py:_on_init():2220] communicating current version
2024-06-19 20:41:53,113 INFO    MainThread:2805280 [wandb_run.py:_on_init():2229] got version response upgrade_message: "wandb version 0.17.2 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2024-06-19 20:41:53,113 INFO    MainThread:2805280 [wandb_init.py:init():804] starting run threads in backend
2024-06-19 20:41:57,877 INFO    MainThread:2805280 [wandb_run.py:_console_start():2199] atexit reg
2024-06-19 20:41:57,877 INFO    MainThread:2805280 [wandb_run.py:_redirect():2054] redirect: wrap_raw
2024-06-19 20:41:57,877 INFO    MainThread:2805280 [wandb_run.py:_redirect():2119] Wrapping output streams.
2024-06-19 20:41:57,877 INFO    MainThread:2805280 [wandb_run.py:_redirect():2144] Redirects installed.
2024-06-19 20:41:57,878 INFO    MainThread:2805280 [wandb_init.py:init():845] run started, returning control to user process
2024-06-19 20:43:11,503 INFO    MainThread:2805280 [wandb_setup.py:_flush():76] Current SDK version is 0.15.12
2024-06-19 20:43:11,504 INFO    MainThread:2805280 [wandb_setup.py:_flush():76] Configure stats pid to 2805280
2024-06-19 20:43:11,504 INFO    MainThread:2805280 [wandb_setup.py:_flush():76] Loading settings from /user/as6154/.config/wandb/settings
2024-06-19 20:43:11,504 INFO    MainThread:2805280 [wandb_setup.py:_flush():76] Loading settings from /user/as6154/dissert/wandb/settings
2024-06-19 20:43:11,504 INFO    MainThread:2805280 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-06-19 20:43:11,504 INFO    MainThread:2805280 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-06-19 20:43:11,504 INFO    MainThread:2805280 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-06-19 20:43:11,504 INFO    MainThread:2805280 [wandb_init.py:_log_setup():528] Logging user logs to /user/as6154/dissert/wandb/run-20240619_204311-xuqf40cx/logs/debug.log
2024-06-19 20:43:11,504 INFO    MainThread:2805280 [wandb_init.py:_log_setup():529] Logging internal logs to /user/as6154/dissert/wandb/run-20240619_204311-xuqf40cx/logs/debug-internal.log
2024-06-19 20:43:11,504 INFO    MainThread:2805280 [wandb_init.py:_jupyter_setup():474] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7fdf5231ed90>
2024-06-19 20:43:11,505 INFO    MainThread:2805280 [wandb_init.py:init():568] calling init triggers
2024-06-19 20:43:11,505 INFO    MainThread:2805280 [wandb_init.py:init():575] wandb.init called with sweep_config: {}
config: {}
2024-06-19 20:43:11,506 INFO    MainThread:2805280 [wandb_init.py:init():593] re-initializing run, found existing run on stack: huj8u14g
2024-06-19 20:43:11,507 INFO    MainThread:2805280 [wandb_run.py:_finish():1934] finishing run hong-exploration/GPT_QFT/huj8u14g
2024-06-19 20:43:11,508 INFO    MainThread:2805280 [wandb_run.py:_atexit_cleanup():2168] got exitcode: 0
2024-06-19 20:43:11,508 INFO    MainThread:2805280 [wandb_run.py:_restore():2151] restore
2024-06-19 20:43:11,509 INFO    MainThread:2805280 [wandb_run.py:_restore():2157] restore done
2024-06-19 20:43:17,087 INFO    MainThread:2805280 [wandb_run.py:_footer_history_summary_info():3599] rendering history
2024-06-19 20:43:17,088 INFO    MainThread:2805280 [wandb_run.py:_footer_history_summary_info():3631] rendering summary
2024-06-19 20:43:17,095 INFO    MainThread:2805280 [wandb_run.py:_footer_sync_info():3558] logging synced files
